pkg/config/orderer.go

```diff
@@ -60,6 +61,32 @@ type Kafka struct {
 	Brokers []string
 }
 
+// EtcdRaft is serialized and set as the value of ConsensusType.Metadata in
+// a channel configuration when the ConsensusType.Type is set to "etcdraft".
+type EtcdRaft struct {
+	Consenters []Consenter
+	Options    EtcdRaftOptions
+}
+
+// Consenter represents a consenting node (i.e. replica).
+type Consenter struct {
+	Host          string
+	Port          uint32
```

Comment:
As mentioned below I ended up changing all address handling to use an Address type with a Host and integer Port because it's easier to string format that where the protos need it than the other way around (parsing the host and port for a url gets somewhat tricky when determining what's valid, if they can specify a protocol at the front etc among other things)